i18n/en/pop_analytics_panel.ftl

```diff
@@ -1,25 +1,33 @@
 hp-analytics-description =
-    Help us improve HP products and Pop!_OS. We use analytics data such as
-    system temperature, battery health, and fan tachometer readings to
-    help us identify ways to make your computer quieter and more performant.
-    Analytics data is shared between HP and Pop!_OS.
+    We use analytics data such as system temperature, battery health, and fan
+    tachometry to improve the performance and efficacy of our devices.
+hp-analytics-prompt =
+    HP may use information about my system to improve HP products and services.
```

Comment:
`hp_vendor_client::purposes(false)` will get purposes without fetching from the API, but still has to call out to `hp-vendor`.

The full purpose data has multiple languages, and has the `purpose_id` and `version` than must be sent to the API to consent.

If using only hard-coded purposes is fine, `hp_vendor_client` could include `purposes.json` directly so this information doesn't require calling out to `hp-vendor`.